iocage_lib/ioc_json.py

```diff
@@ -110,6 +110,34 @@ class IOCZFS(object):
     def __init__(self):
         self.zfs = libzfs.ZFS(history=True, history_prefix="<iocage>")
 
+    @property
+    def iocroot_path(self):
+        # For now we assume iocroot is set, however moving on we need to think
+        # how best to structure this
+        # We should give thought to how we handle
+        for pool in self.pools:
```

Comment:
This only returns the first instance, which on a misconfigured system could be more than 1